docs/mast/mast.rst

```diff
@@ -813,13 +808,35 @@ ccd a target pixel file will be produced for each one.
                   2  APERTURE      1 ImageHDU        80   (5, 5)   int32
 
 
-The `~astroquery.mast.TesscutClass.download_cutouts` function takes a coordinate or object name
-(such as "M104" or "TIC 32449963") and cutout size (in pixels or an angular quantity) and
-downloads the cutout target pixel file(s).
+Requesting a cutout by moving_target accesses the
+`MAST Moving Target TESScut API <https://mast.stsci.edu/tesscut/docs/getting_started.html#moving-target-cutouts>`__
+and returns a target pixel file, with format described
+`here <https://astrocut.readthedocs.io/en/latest/astrocut/file_formats.html#path-focused-target-pixel-files>`__.
+The moving_target may be any object name or ID understood by the
+`JPL Horizonf ephemerades interface <https://ssd.jpl.nasa.gov/horizons.cgi>`__.
+
+.. code-block:: python
+
+                >>> from astroquery.mast import Tesscut
```

Comment:
I run into a timeout with this example

Reply:
@jaymedina Try this example now, if I remember correctly the JPL horizons service was having timeout problems last August, so that might have been the problem.

Reply:
Looks good

```
In [3]: hdulist = Tesscut.get_cutouts(moving_target="Eleonora", size=5, sector=6
   ...: )

In [4]: hdulist[0].info()
Filename: <class '_io.BytesIO'>
No.    Name      Ver    Type      Cards   Dimensions   Format
  0  PRIMARY       1 PrimaryHDU      54   ()      
  1  PIXELS        1 BinTableHDU    150   355R x 16C   [D, E, J, 25J, 25E, 25E, 25E, 25E, J, E, E, 38A, D, D, D, D]   
  2  APERTURE      1 ImageHDU        97   (2136, 2078)   int32   
  ```

Reply:
Looks like we have 355 rows now instead of 356. Must be a numpy thing?

Reply:
@ceb8 would you mind giving me writing access to your forked astroquery repo? I have some updates I need to commit to this branch.

Reply:
@jaymedina you should have an invite now